utils/lib/Quickstart.ts

```diff
@@ -256,16 +268,17 @@ class Quickstart {
    * Static method that returns a list of every quickstarts
    * @returns - A list of all quickstarts
    */
-  static getAll(basePath?: string): Quickstart[] {
+  static getAll(basePath?: string, context?: QuickstartContext ): Quickstart[] {
```

Comment:
Two optional parameters in a row leads to some weird call signatures when you're calling the function, such as `getAll(undefined, {})` which is really confusing without hover docs. Since these are both optional, why don't we convert it to an `options` object argument with the fields `basePath` and `context`